website/source/docs/platform/k8s/helm/examples.html.md

```diff
@@ -21,7 +21,7 @@ The below `values.yaml` can be used to set up a single server Vault cluster with
 ```yaml
 global:
   enabled: true
-  image: "vault:1.2.4"
+  image: "vault:1.3.0"
```

Comment:
This needs to be `1.3.1`

Reply:
This actually got updated underneath this PR so the merge from master fixed this.